### DISCUSSION TITLE
(原因がわかっている) エラーが何度も続くだけでサポートへの問い合わせを要求しないでほしい

### DISCUSSION THREAD

**Discussion by mikecat** [increments/qiita-discussions#878] [要望・アイデア・不具合]
記事を書いていると、このようなメッセージが出ました。

![問題のメッセージ](https://github.com/increments/qiita-discussions/assets/6667599/0c33aa23-2b26-471d-9926-a5c5bec6fe5c)

確認すると、メンテナンスを実施していることが原因らしいことがわかりました。

[XユーザーのQiita (キータ) 公式さん: 「【📣メンテナンス実施のお知らせ】 これよりメンテナンスを実施いたします。 終了時刻は 2024/07/07 10:00 頃を予定しております。 メンテナンス時間中は Qiita にアクセスすることが出来ません🙇 ご迷惑をお掛けいたしますがよろしくお願いいたします。 https://t.co/lwrP6NFEXv」 / X](https://x.com/Qiita/status/1809724076178694171)

とはいえ、記事を書き進めると、同じメッセージが繰り返し出てしまいます。  
そのため、サポートに問い合わせるべきということになってしまいます。

しかし、今回の場合はメンテナンスが原因であるらしいことがわかるため、サポートへの問い合わせは行わないべきだと思います。  
同様の現象は、Qiita のメンテナンス以外にも、クライアント側の回線の切断 (オフライン作業) やメンテナンスなどでも発生する可能性があります。

ただ単に「何度も続く場合」ではなく、「予期せず」などの条件を追加し、余計だと思える問い合わせを要求しないでもらいたいです。

**Comment by tanaka-asami415** [comment]
@mikecat 
ご意見ありがとうございます。
この度はご不便をおかけして申し訳ございません。
メッセージ内容の変更を検討いたします🙇
